awxkit/awxkit/api/pages/api.py

```diff
@@ -186,7 +186,7 @@ def _filtered_list(self, endpoint, value):
             return endpoint.get(id=int(value))
         options = self._cache.get_options(endpoint)
         identifier = next(field for field in options['search_fields'] if field in ('name', 'username', 'hostname'))
-        return endpoint.get(**{identifier: value})
+        return endpoint.get(**{identifier: value},all_pages=True)
```

Comment:
That's the ticket, but the lack of a space is going to cause the linting check to fail

Reply:
My bad, pushed it again with proper indentation